torchmultimodal/models/video_vqvae.py

```diff
@@ -16,40 +16,37 @@
 
 
 def video_vqvae(
-    in_channel_dim: int = 3,
-    encoder_hidden_dim: int = 240,
-    encoder_kernel_size: int = 3,
-    encoder_stride: int = 2,
-    encoder_n_layers: int = 1,
-    n_res_layers: int = 4,
-    attn_hidden_dim: int = 240,
-    num_embeddings: int = 1024,
-    embedding_dim: int = 256,
-    decoder_hidden_dim: int = 240,
-    decoder_kernel_size: int = 4,
-    decoder_stride: int = 2,
-    decoder_n_layers: int = 1,
+    in_channel_dim: int,
+    encoder_hidden_dim: int,
+    encoder_kernel_size: int,
```

Comment:
Just to confirm: this API does not support cases where kernel_size, strides are different layer/dim-wise, right?